main/integration-tests/__test__/api-tests/studies/update-study-api.test.js

```diff
@@ -0,0 +1,78 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const _ = require('lodash');
+const axios = require('axios').default;
+const UpdateStudyFixture = require('./__fixtures__/update-study-fixture');
+const { getTestAdminToken } = require('../../../utils/auth-tokens');
+const { listStudies } = require('../../../utils/studies');
+const { updateStudyParams, getStudyParams } = require('../../../helpers/api-param-generator');
+
+describe('Update Study Test', () => {
+  let testFixture;
+
+  beforeAll(async () => {
+    testFixture = new UpdateStudyFixture();
+
+    if (!UpdateStudyFixture.baseReady) {
+      await testFixture.setupParent();
+    }
+    if (!UpdateStudyFixture.ready) {
+      await testFixture.setupPreRequisites();
+    }
+  });
+
+  describe('Update Study API', () => {
+    it('should fail while trying to update Open Data studies', async () => {
+      // BUILD
+      const adminBearerToken = await getTestAdminToken(testFixture.testConfig);
```

Comment:
There's a lot of passing bearer token around as argument to different functions. You can create an axios instance with fixed headers.

See: https://github.com/awslabs/fhir-works-on-aws-deployment/blob/mainline/integration-tests/utils.ts#L43

I suggest having a function like:
```
const getSWBClient = (role) => {//returns axios instance with headers for user with that role}
```

Reply:
Done